platform-common-string/src/main/java/com/softicar/platform/common/string/regex/PatternFinder.java

```diff
@@ -11,52 +11,80 @@
  */
 public class PatternFinder {
```

Comment:
All changes in this class are API-compatible.
Methods were re-ordered, and a new method was added, to enable use of a pre-compiled pattern.